src/components/POLine/POLineView.js

```diff
@@ -242,6 +240,8 @@ class POLineView extends Component {
     const isWorkflowStatusOpen = order.workflowStatus === WORKFLOW_STATUS.open;
     const checkInLocation = isWorkflowStatusOpen ? `${checkinURL}/items` : `${checkinURL}/history`;
     const receivingLocation = isWorkflowStatusOpen ? receivingURL : `${receivingURL}-history`;
+    const estimatedPrice = get(line, ['cost', 'poLineEstimatedPrice'], 0);
+    const fundDistributions = get(line, ['fundDistribution'], []);
```

Comment:
No needed extra brackets for `'fundDistribution'`

Reply:
fixed